### PR TITLE
Pinyin "ion" trans to "iong"

### PR DESCRIPTION
**Pull request by san-Hei**
图片

### CODE REVIEW

**Comment by wengxt** [comment]
I think this one doesn't need to have a standalone entry in the config. Maybe we can rename NG_GN to CommonTypo and make then use the same flag.

**Comment by san-Hei** [comment]
ok, I change it

**Comment by san-Hei** [comment]
done

File: src/libime/pinyin/pinyinencoder.h [hunk 1]
```diff
@@ -23,7 +23,7 @@ class ShuangpinProfile;
 
 enum class PinyinFuzzyFlag {
     None = 0,
-    NG_GN = 1 << 0,
+    CommonTypo = 1 << 0,
```
Comment (wengxt) [inline src/libime/pinyin/pinyinencoder.h:26]: Please Also keep NG_GN as an old name for compatiblity.
E.g.
```
 NG_GN  [[deprecated]] = CommonTypo, 
```

**Pull request merged**